GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockVirtualizationInstance.cs

```diff
@@ -27,7 +30,11 @@ public MockVirtualizationInstance()
             this.WriteFileReturnResult = HResult.Ok;
         }
 
-        public HResult CompletionResult { get; set; }
+        public HResult CompletionResult
+        {
+            get { return this.completionResult; }
+            set { this.completionResult = value; }

```

Comment:
It looked like this can be set as `private`.  Not sure how many other properties could be as well but this is a mock for testing so probably doesn't matter too much.

Reply:
Agreed, and since it can be private I'll just remove the property and we can use `this.completionResult` directly in the mock